.github/actions/cargo_target_dir_pre_cache/action.yml

```diff
@@ -7,12 +7,20 @@ runs:
       run: |
         crates=($(cargo metadata --no-deps --format-version 1 | jq -r '.workspace_members[] | split(" ")  | .[0] | gsub("-";"_")'))
 
+        # Get target folder size
+        du -sh target
+
+        rm -rf ./target/debug/examples
+        rm -rf ./target/debug/incremental
```

Comment:
We are deleting examples and incremental folders as they consist of Ockam example and test build outputs and will be recompiled